edn_format/edn_lex.py

```diff
@@ -174,9 +175,9 @@ def t_WHITESPACE(t):
 
 
 def t_CHAR(t):
-    # from "!" to "~" = all printable ASCII chars except the space
-    r"(\\[!-~])"
-    t.value = t.value[1]
+    # uXXXX hex code or from "!" to "~" = all printable ASCII chars except the space
```

Comment:
This comment should be changed as it doesn’t reflect the code below, because of the addition of `\w` in the range.

Reply:
Right, missed that